SwiftBetaXcode13/ContentView.swift

```diff
@@ -9,8 +9,13 @@ import SwiftUI
 
 struct ContentView: View {
     var body: some View {
-        Text("Hello, world!")
-            .padding()
+        VStack {
+            Text("Hello, world!")
+                .padding()
```

Comment:
Nice! maybe you can use 
```suggestion
                .padding(20)
```